src/apps/util/config.clj

```diff
@@ -173,6 +173,16 @@
   [props config-valid configs]
   "apps.workspace.metadata.communities.attr" "cyverse-community")
 
+(cc/defprop-optstr workspace-metadata-certified-apps-attr
+  "A metadata attr that can be applied to an app that has been reviewed and certified by DE administrators."
+  [props config-valid configs]
+  "apps.workspace.metadata.certified.attr" "cyverse-blessed")
```

Comment:
I wonder if we should prevent the non-admin service calls in https://github.com/cyverse-de/apps/blob/master/src/apps/metadata/avus.clj from setting an AVU with this attribute.

Reply:
Ah, that's a good idea! I'll add it, thanks!

Reply:
It looks like there's no validation for the `beta` and `communities` attributes either. I'll add checks for those as well unless anyone can think of a reason not to.

Reply:
Good catch! Sounds like a good idea to me 👍